app/assets/javascripts/oxalis/model/helpers/nml_helpers.js

```diff
@@ -140,10 +141,22 @@ function serializeMetaInformation(state: OxalisState, buildInfo: APIBuildInfoTyp
   ]);
 }
 
+function serializeBoundingBox(bb: BoundingBoxType, name: string): string {
+  return serializeTag(name, {
+    topLeftX: bb.min[0],
+    topLeftY: bb.min[1],
+    topLeftZ: bb.min[2],
+    width: bb.max[0] - bb.min[0],
```

Comment:
Can you use `convertFrontendBoundingBoxToServer` from `reducer_helpers.js` here like this?

```
const [topLeft, width, height, depth] = convertFrontendBoundingBoxToServer(bb);
const [topLeftX, topLeftY, topLeftZ] = topLeft;
return serializeTag(name, { topLeftX, topLeftY, topLeftZ, width, height, depth });
```

Then, we don't duplicate the `max - min` logic :)

Reply:
Ah, I see, you moved the code. But maybe it's a good idea, anyway, to reduce the boundingbox-related code a bit.